debian/control

```diff
@@ -1,36 +1,36 @@
-Source: livechart
+Source: livechart2
 Section: libs
 Priority: optional
 Maintainer: elementary, Inc. <builds@elementary.io>
 Build-Depends: debhelper (>= 10.5.1),
                libgee-0.8-dev,
                libglib2.0-dev (>= 2.50),
-               libgtk-3-dev (>= 3.22),
+               libgtk-4-dev (>= 4.12.0),
                valac,
                meson
 Standards-Version: 4.5.0
-Vcs-Git: https://github.com/lcallarec/live-chart.git
-Vcs-Browser: https://github.com/lcallarec/live-chart
-Homepage: https://github.com/lcallarec/live-chart
+Vcs-Git: https://github.com/elementary/live-chart.git
+Vcs-Browser: https://github.com/elementary/live-chart
+Homepage: https://github.com/elementary/live-chart
```

Comment:
Maybe we may want to split these commits that replaces URLs into different PRs so that we can cherry-pick that commit to the `debian-packaging-1.9.x` branch too.

Reply:
Fixed in #13